### PR TITLE
Added text exports

### PR DESCRIPTION
**Pull request by antoKeinanen**
Added options to export in the following formats:
- copy to clipboard
- export as text file

These features were originally marked as ToDo.

### CODE REVIEW

File: src/utils/export.js [hunk 1]
```diff
@@ -46,11 +46,42 @@ export default async (format) => {
...
+        let parsedData = ""
+        data.forEach((line) => {
+            const strippedLine = line.replace(/<[^>]*>/g, "")
+            parsedData = `${parsedData}\n${atob(strippedLine)}`
+        })
+        parsedData = parsedData.slice(1)
+
+        console.debug(parsedData)
+
+        const element = document.createElement("a")
```
Comment (Esinko) [inline src/utils/export.js:78]: Consider using the `saveAs` function for this.

File: src/utils/export.js [hunk 2]
```diff
@@ -46,11 +46,42 @@ export default async (format) => {
...
+
+        data.shift()
+        let parsedData = ""
+        data.forEach((line) => {
+            const strippedLine = line.replace(/<[^>]*>/g, "")
+            parsedData = `${parsedData}\n${atob(strippedLine)}`
+        })
+        parsedData = parsedData.slice(1)
+
+        console.debug(parsedData)
```
[the remark this answers is not in the document]
Comment (antoKeinanen) [reply src/utils/export.js]: fixed

File: src/utils/export.js (hunk 1 above)
Comment (antoKeinanen) [reply src/utils/export.js]: fixed

**Comment by Esinko** [comment]
LGTM

**Pull request merged**